zetaclient/chains/ton/observer/inbound.go

```diff
@@ -19,32 +19,37 @@ import (
 )
 
 const (
-	// MaxTransactionsPerTick is the maximum number of transactions to process on a ticker
-	MaxTransactionsPerTick = 100
+	// maximum number of transactions to process on a ticker
+	maxTransactionsPerTick = 100
```

Comment:
Wouldn't it need to be customizable depending on localnet/testnet/mainnet?

If so we should create an issue to make it a parameter